ReleaseNotes.md

```diff
@@ -11,6 +11,19 @@ additional questions or comments.
 
 Note : The agent version(s) below has dates (ciprod\<mmddyyyy\>), which indicate the agent build dates (not release dates)
 
+### 01/09/2023 -
+##### Version microsoft/oms:ciprod01092023-864ecd78 Version mcr.microsoft.com/azuremonitor/containerinsights/ciprod:ciprod01092023-864ecd78 (linux)
+##### Version microsoft/oms:win-ciprod01092023-864ecd78 Version mcr.microsoft.com/azuremonitor/containerinsights/ciprod:win-ciprod01092023-864ecd78 (windows)
+##### Code change log
+- Linux Agent
+  - Replace ifconfig.co url with mcr.microsoft.com for network connectivity test
+  - Add xvda in devices so that telegraf can fetch the Disk IO data from xvda disk as well
+- Common
```

Comment:
Add the other item under common

- Implementation of Data collection settings for Cost optimization feature